features/location.yml

```diff
@@ -0,0 +1,22 @@
+name: Location
+description: "The `Location` global object represents the current address of the web page loaded in the browser. It makes all component parts of the address accessible and can be manipulated to initiate navigations."
+spec: https://html.spec.whatwg.org/multipage/nav-history-apis.html#the-location-interface
```

Comment:
This is… the most intense warning I've ever seen in a web spec. 😅

Reply:
I know, right?!